platform/darwin/test/MGLStyleLayerTests.h

```diff
@@ -17,3 +15,11 @@
 @property (nonatomic, readonly, copy) NSString *lemma;
 
 @end
+
+@interface NSValue (MGLStyleLayerTestAdditions)
+
++ (instancetype)valueWithMGLVector:(CGVector)vector;
```

Comment:
macOS lacks `+[NSValue valueWithCGVector:]` and `NSValue.cgVectorValue` for some reason. It’s possible to call `+[NSValue value:withObjCType:]` and `-[NSValue getValue:]` instead, but that approach requires more than one statement at a time, so it doesn’t fit neatly into the assertion statements.

I’m considering upstreaming this category to the SDK proper, using `+[NSValue valueWithMGLCoordinate:]` and `NSValue.mglCoordinateValue` as a precedent.